### ISSUE TITLE
Proposed question: improve accuracy of a vanilla CNN model

### ISSUE THREAD

**Issue by samhubs** [Exorust/TorchLeet#7]
When I completed the m2 problem, I also wanted to check if there were ways to improve accuracy from ~69% to higher. The idea that I explored was to creage multiple iteration of m2 solution CNN model (let's call it vanilla) without major changes: added BatchNorm, Dropout, Weight initialization, Scheduler, added additional Conv and Linear layers. I  am convinced there should be a question where we provide a base vanilla CNN model and let the learners play with pytorch options and try to improve the accuracy. I am not clear on how to frame the question and the solution because I tried 8-10 iterations and the solution notebook kept on growing without much improvement :). I know using a models.ResNet50 would do the trick but I wanted to extract the most juice from the vanilla CNN model. 

Thanks,

**Comment by Exorust** [comment]
Makes perfect sense. Would you like to raise a PR to add your changes as a bonus? :)

**Comment by samhubs** [comment]
Yes, will take care of it this week. Thanks,
